app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx

```diff
@@ -158,7 +159,7 @@ describe("The Create Project page", () => {
     pageTestingHelper.loadQuery({
       Form() {
         return {
-          jsonSchema: reportingRequirementProdSchema,
+          jsonSchema: projectContactProdSchema,
```

Comment:
I ended up not needing to delve into mock resolver context here, but I added a link to the docs for it in the other db-fetching cards